### ISSUE TITLE
must set udp timeout

### ISSUE THREAD

**Issue by raintears2** [semigodking/socks5chk#1]
req = b"\x12\x34\x01\x00\x00\x01\x00\x00\x00\x00\x00\x00\x05\x62\x61\x69\x64\x75\x03\x63\x6f\x6d\x00\x00\x01\x00\x01"
        

        s.settimeout(4)
        s.sendto(req, ("8.8.8.8", 53))
        
        (rsp, address)= s.recvfrom(4096)
        
        if rsp[0] == req[0] and rsp[1] == req[1]:
            print("UDP check passed")
        else:
            print("Invalid response")
        s.settimeout(None)

**Comment by semigodking** [comment]
Yes, you are.right. Could you please submit a pr for timeout? Thank you.
